Keyed/Messages.xml

```diff
@@ -57,9 +57,9 @@
   <MessageOutOfNearbyShellsFor>{PAWN_labelShort} nie może znaleźć amunicji do {GUN_label}.</MessageOutOfNearbyShellsFor>
 
   <!-- EN: Warning: {ANIMAL_kindBasePlural} have a {1} chance to attack when harmed from rifle range. Chance is higher if closer, less with good hunter stealth. -->
-  <MessageAnimalsGoPsychoHunted>Uwaga: Jest {1} szans na to, że {ANIMAL_kindBasePlural} zaatakują, gdy zrani się je z dystansu. Szansa jest większa jeśli cel jest bliżej, zmniejsza się wraz z umiejętnością kamuflażu myśliwego.</MessageAnimalsGoPsychoHunted>
+  <MessageAnimalsGoPsychoHunted>Uwaga: Jest {1} szans na to, że {ANIMAL_kindBasePlural} zaatakuje, gdy zrani się je z dystansu. Szansa jest większa jeśli cel jest bliżej, zmniejsza się wraz z umiejętnością kamuflażu myśliwego.</MessageAnimalsGoPsychoHunted>
```

Comment:
Hmm, tutaj jest powrót to tej błędnej wersji. 

Powinno być jak poprzednio:
Jest X szans na to, że {ANIMAL_kindBasePlural = DZIKI, SCZCZURY} **zaatakują,**